src/content/1.7/development/webservice/tutorials/1-creating-access.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Provide access to the Webservice
-menuTitle: Provide access
-weight: 3
+title: 1 - Creating access to the Webservice
```

Comment:
AFAIK SEO people recommend having the same text for the page title and first H1. Not sure if relevant. If needed, just remove the "1 - " from the title because it doesn't add anything.

Reply:
The number in the title is also used when displaying the children, you think we should remove them for sub pages
Do we keep it on the first level folders? (1 - Creating access to the Webservice, 2 - Testing access to the Webservice, ...)

Reply:
Maybe add an option to the `{{% children %}}` short tag so that it uses `<ol>` instead  of `<ul>`?

Like this: `{{% children list="ol" %}}`

[See source](https://github.com/PrestaShop/docs/blob/master/src/themes/hugo-theme-learn/layouts/shortcodes/children.html)

Reply:
Great idea! I added the option in children, and in addition with the use of menuTitle we have the right balance I think

Title, h1, urls and file names don't include the numbers

But displayed children, and titles in the menu have, and we can safely change the menu title if needed as it won't have any drawback on SEO (just the link text changes in the menu)

Reply:
👍